### PR TITLE
Tracking app version where user created account

### PR DESCRIPTION
**Pull request by sendhil**
cc @diegoreymendez , @meremagee

### CODE REVIEW

File: WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m [hunk 1]
```diff
@@ -204,6 +204,10 @@ - (void)trackMixpanelDataForInstructions:(WPAnalyticsTrackerMixpanelInstructions
     [instructions.peoplePropertiesToAssign enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
         [self setValue:obj forPeopleProperty:key];
     }];
+    
+    [instructions.superPropertiesToAssign enumerateKeysAndObjectsUsingBlock:^(id key, id obj, BOOL *stop) {
```
Comment (sendhil) [inline WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m]: I'm aware we want to change this in the future, but for now let's hold off on it until the next release as I was trying to keep this change more minimal given how late we are in the cycle.

Comment (diegoreymendez) [inline WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m]: :+1:

File: WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanelInstructionsForStat.m [hunk 2]
```diff
@@ -81,4 +83,11 @@ - (void)setPeopleProperty:(NSString *)property toValue:(id)value
     _peoplePropertiesToAssign[property] = value;
 }
 
+- (void)setSuperProperty:(NSString *)property toValue:(id)value
+{
+    NSParameterAssert(property != nil);
+    NSParameterAssert(value != nil);
```
Comment (diegoreymendez) [inline WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanelInstructionsForStat.m]: Good call on the asserts.

**Comment by diegoreymendez** [comment]
:shipit:

**Pull request merged**